packages/landing-gatsby/src/containers/App/Banner/index.js

```diff
@@ -132,18 +132,17 @@ DomainSection.defaultProps = {
     ml: 'auto',
   },
   title: {
-    content: 'Prozesses and tools for selfmanaged and distributed teams',
+    content: 'Processes and tools for selfmanaged and distributed teams',
     fontSize: ['18px', '24px', '24px', '30px', '34px'],
     fontWeight: '700',
     color: '#091632',
     letterSpacing: '1.2px',
     lineHeight: '42px',
     mb: '10px',
-    mt: '15px',
+    mt: '160px',
   },
   description: {
-    content:
-    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam.',
+    content: 'Teal Garden is a collection of workflows to enhance performance & productivity of teams which manage themselves independently. Join our community of organisation architects!',
```

Comment:
Needs to be shorter and clear CTA:
Become part of the future of digital collaboration. Join our community of teal organisation architects.